src/Services/RefundGateway/RefundGateway.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tipoff\Refunds\Services\RefundGateway;
+
+use Tipoff\Locations\Models\Location;
+use Tipoff\Payments\Enums\Gateway;
+use Tipoff\Support\Contracts\Payment\ChargeableInterface;
+use Tipoff\Support\Contracts\Services\BaseService;
+
+interface RefundGateway extends BaseService
```

Comment:
Added so payments/refunds follow same implementation pattern with use of gateway binding to perform the requested operation.